quiz.css

```diff
@@ -0,0 +1,33 @@
+.card {		border: 1px solid lightgray;
```

Comment:
Style requirement 1 asks for the border to be lightblue, not lightgray. However, I definitely dig the box-shadows. Nice touch!

On a side note, I would recommend being more consistent with your spacing and my more generous with the spacing too (not required). It looks like you were going for a compact look, but you want it to be readable and easily editable. There are tools you can use later to squish and minify your code for optimization.